mqt/ddsim/qasmsimulator.py

```diff
@@ -7,19 +7,53 @@
 from typing import Dict, List, Union
 
 from qiskit import QiskitError, QuantumCircuit
-from qiskit.compiler import assemble
-from qiskit.providers import BackendV1, Options
+from qiskit.circuit import Parameter
+from qiskit.circuit.library import (
+    MCMT,
+    GlobalPhaseGate,
+    MCPhaseGate,
+    MCXGrayCode,
+    MCXRecursive,
+    MCXVChain,
+    Measure,
+    RXGate,
+    RYGate,
+    RZGate,
+)
+from qiskit.providers import BackendV2, Options
 from qiskit.providers.models import BackendConfiguration, BackendStatus
-from qiskit.qobj import PulseQobj, QasmQobj, QasmQobjExperiment, Qobj
+from qiskit.qobj import PulseQobj, QasmQobj
 from qiskit.result import Result
+from qiskit.transpiler import Target
 
 from mqt.ddsim import CircuitSimulator, __version__
 from mqt.ddsim.job import DDSIMJob
 
 logger = logging.getLogger(__name__)
```

Comment:
I know its not code from you, but since the logger is not used anywhere, please remove it.